react-components/src/apps/details/ToolDetails.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @author psarando
+ */
+import React, { Component } from 'react';
+import {
+    MediaCard,
+    MediaCardGroup,
+    SummaryText,
+} from "cyverse-ui";
+
+class ToolDetails extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedToolIndex: 0
+        };
+
+        // This binding is necessary to make `this` work in the callback
+        this.onToolSelectionChange = this.onToolSelectionChange.bind(this);
+    }
+
+    onToolSelectionChange(event, selectedToolIndex) {
+        this.setState({
+            selectedToolIndex: selectedToolIndex
+        });
+    }
+
+    render() {
+        let tools      = this.props.app.tools,
+            appearance = this.props.appearance,
+            labelClass = appearance.css().label(),
+            valueClass = appearance.css().value();
+
+        return (
+            <MediaCardGroup>
+                {
+                    tools.map( (toolInfo, index) => (
+                <MediaCard
+                    key={index}
+                    style={{width: '100%', border: "1px solid black"}}
```

Comment:
Just a discussion, not really a comment on your PR - how strict do we want to be with putting stylings in a GWT appearance?
I feel like any text strings should be in the GWT appearance because GWT does that whole internationalization thing and then we can share appearances (if needed) between any GWT views and React views while we're moving over piece by piece. But I wonder if `style` modifications and CSS files should live in React in some way?  I kind of like keeping views and appearances separate in any case so it's easy to work in "themes" if needed, but I'm open to others' ideas.

Reply:
That's a good point. I threw this into the media cards to see if it fixed some styling issues and just forgot about it. There was some "CSS in Javascript" hype not too long ago, but I think we should be consistent in our app, which means keeping styles in the appearance. There might be cases where it's not practical to get a style from a className in an appearance, but we should probably try.

I'll try to move this style into an appearance before merging 👍

Reply:
So are sticking with `css` and `string` in appearance ?

Reply:
That seems best for now: let themes and GWT handle it.